src/main/java/frc/robot/constants/DriveConstants.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.constants;
+
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+
+public class DriveConstants {
+
+    public static final double ksVolts = 0.046401;
+    public static final double kvVoltSecondsPerMeter = 0.14137;
+    public static final double kaVoltSecondsSquaredPerMeter = 0.0068514;
+    public static final DifferentialDriveKinematics kDriveKinematics = new DifferentialDriveKinematics(.68);
```

Comment:
```suggestion
    public static final double kDriveTrackWidthMeters = 0.68;
    public static final DifferentialDriveKinematics kDriveKinematics = new DifferentialDriveKinematics(kDriveTrackWidthMeters);
```
DifferentialDriveKinematics handles most of the times we would want to know the track width but its sometimes helpful to be able to access track width directly.